src/Swarm/Game/Step.hs

```diff
@@ -913,12 +919,52 @@ execConst c vs s k = do
         -- set the copied robot to be a "system" robot so it is
         -- capable of executing any commands; the As command
         -- already requires "God" capability.
-        v <-
-          evalState @Robot (r & systemRobot .~ True) . evalState @GameState g $
-            runCESK (Out prog s [FApp (VCApp Force []), FExec])
+
+        -- -----------------------------------------------------------------------------
+        -- TODO: move this code out
+        -- Slightly edited copy of Unit.hs version of runCESK:
+        let runCESK' :: CESK -> StateT Robot (StateT GameState IO) (Either Text Value)
```

Comment:
Don't we already have this function in `Step.hs`?  Hmm, or maybe that's only in my `challenge-mode` branch?  Feel free to copy it: https://github.com/byorgey/swarm/pull/285/files#diff-bb4a5ea570cea9f815cdd1d92b33d4b6bd8daaa7d74c7014916c642c159e7d80R147-R173

Reply:
Thanks, I will do so during rebase :slightly_smiling_face: